saleor/payment/gateways/adyen/tests/webhooks/test_handle_notifications.py

```diff
@@ -239,6 +239,48 @@ def test_handle_authorization_with_adyen_auto_capture(
     assert external_events.count() == 1
 
 
+@patch("saleor.payment.gateway.refund")
+def test_handle_authorization_with_adyen_auto_capture_and_inactive_payment(
+    refund_mock, notification, adyen_plugin, inactive_payment_adyen_for_checkout
+):
+    payment = inactive_payment_adyen_for_checkout
+    payment_id = graphene.Node.to_global_id("Payment", payment.pk)
+
+    notification = notification(
+        merchant_reference=payment_id,
+        value=price_to_minor_unit(payment.total, payment.currency),
+    )
+
+    plugin = adyen_plugin(adyen_auto_capture=True)
+    handle_authorization(notification, plugin.config)
+
+    payment.refresh_from_db()
+
+    assert payment.charge_status == ChargeStatus.FULLY_CHARGED
+    assert refund_mock.called
```

Comment:
Could you use blank lines or `given, when and then` annotation in code? Right now is hard to find what are you testing.

Reply:
Could you update also other tests? I see similar situations in other tests in `saleor/payment/gateways/adyen/tests/webhooks/test_handle_notifications.py` and `saleor/payment/gateways/stripe/tests/test_webhooks.py`.